userbenchmark/functorch/__init__.py

```diff
@@ -0,0 +1,31 @@
+import torch
+from ..utils import dump_output
+from .cases import benchmark_cases
+from .util import benchmark
+import pprint
+
+
+BM_NAME = 'functorch'
+
+
+def run_benchmarks():
+    metrics = {}
+
+    for case_ctor in benchmark_cases:
+        case = case_ctor()
+        runtime_ms = benchmark(case)
+        metrics[case.name()] = runtime_ms
+    return metrics
+
+
+def run(args):
```

Comment:
I don't know what `args` should be, this file currently ignores it.

Reply:
This is part of the interface between the benchmark driver and userbenchmark, because some benchmarks (e.g. distributed) want to pass in arguments. It is totally fine that functorch doesn't need extra arguments to run, but we suggest to annotate the `args` parameter to `List[str]`.